css/style.css

```diff
@@ -1,15 +1,458 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	overflow-x: hidden;
+}
+
 body {
 	font-family: 'GothamPro Bold';
+	margin: 0;
+	padding: 0;
 }
 
 a {
 	text-decoration: none;
+	display: inline-block;
+}
+
+/*header start*/
+.header-content {
+	max-width: 1170px;
+	margin: 0 auto;
 }
 
-.header__container {
+.header-container {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
 	max-width: 1030px;
+	padding: 0 15px;
+	height: 115px;
+	margin: 0 auto;
+}
+
+.header-container__inner {
+	flex: 1;
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+.header-logo {
+	width: 55px;
+	height: 55px;
+	margin-right: 46px;
+	margin-top: 2px;
+}
+
+.header-mobile__logo {
+	display: none;
+}
+
+.header-menu__humb {
+	display: none;
+}
+
+.header-logo__logo {
+	width: 100%;
+}
+
+.header-menu__list li {
+	list-style-type: none;
+	display: inline-block;
+	margin: 0;
+	padding: 0;
+}
+
+.header-menu__list{
+	white-space: nowrap;
+	margin: 0;
+	padding: 0;
+}
+
+.header-menu__link {
+	color: #000;
+	display: inline-block;
+	margin: 0;
+	padding: 1px 30px 0 0;
+	font-size: 12px;
+}
+
+.header-menu__list li:last-child {
+	padding-right: 0;
+}
+
+.header-container__wrap {
+	flex: 1;
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	white-space: nowrap;
+	padding-top: 2px;
+}
+
+.header-search {
+	margin-right: 34px;
+}
+
+.header-search .header-search__form {
+	border-bottom: 1px solid #000;
+}
+
+.header-search__form {
+	position: relative;
+	max-width: 178px;
+}
+
+.header-search__search , 
+.header-search__btn {
+	border: none;
+	outline: none;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457135)